### PR TITLE
Always pass focus props to setFocus

### PR DESCRIPTION
**Pull request by ellatrix**
This fixes an issue where the toolbar does not hide when typing in a quote block or table. Somehow I don't like that this is the responsibility of the block implementor but yeah. :)

### CODE REVIEW

File: blocks/library/code/index.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ registerBlockType( 'core/code', {
 		return (
 			<TextareaAutosize
 				value={ attributes.content }
-				onFocus={ setFocus }
+				onFocus={ () => setFocus( {} ) }
```
Comment (iseulde) [inline blocks/library/code/index.js]: Avoid passing a `SyntheticEvent`.

Comment (aduth) [inline blocks/library/code/index.js]: >Avoid passing a `SyntheticEvent`.

Should this be added as an inline comment?

**Comment by aduth** [comment]
>Somehow I don't like that this is the responsibility of the block implementor but yeah. :)

Related: #878

**Review by aduth — APPROVED** [review]

File: blocks/library/code/index.js (hunk 1 above)
Comment (iseulde) [inline blocks/library/code/index.js]: After 3f65f3c632646c2cee83639c6efc1455fea217bf, it doesn't even seem necessary anymore.

**Comment by aduth** [comment]
Do you need a fresh review here? Unclear if anything's changed since my last approval.

**Comment by iseulde** [comment]
Sorry, I left a comment (https://github.com/WordPress/gutenberg/pull/1027#discussion_r120615964) and made a commit.

**Review by aduth — APPROVED** [review]
Still looks good 👍

**Pull request merged**